Packs/Core/Playbooks/playbook-Get_entity_alerts_by_MITRE_tactics.yml

```diff
@@ -82,32 +67,138 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-  "2":
-    id: "2"
-    taskid: 0491050a-7fb5-408d-8b73-37a5546c6309
+  "64":
+    id: "64"
+    taskid: e2587a6e-11b1-488d-8fd5-2a2132443c5f
     type: condition
     task:
-      id: 0491050a-7fb5-408d-8b73-37a5546c6309
+      id: e2587a6e-11b1-488d-8fd5-2a2132443c5f
       version: -1
-      name: Should hunt for Persistence techniques?
-      description: Whether to hunt for Persistence techniques.
+      name: Should hunting be executed?
       type: condition
       iscommand: false
       brand: ""
     nexttasks:
       '#default#':
-      - "4"
+      - "114"
       "yes":
-      - "14"
+      - "68"
     separatecontext: false
     conditions:
     - label: "yes"
       condition:
       - - operator: isEqualString
           left:
             value:
-              complex:
-                root: inputs.HuntPersistenceTechniques
+              simple: inputs.RunAll
```

Comment:
Suggestion - 
When RunAll is true, just add a task with the full query and save the execution of multiple tasks that builds the query. you can use "SetIfEmpty" in the query execution task and skip the query builders tasks when RunAll is true.

Reply:
done to both playbooks, if runall is enabled we will just execute the full query instantly